lib/browser.js

```diff
@@ -121,15 +121,26 @@ class Browser {
 
   reconnect (newSocket) {
     if (this.state === EXECUTING_DISCONNECTED) {
+      // In case the socket just lost connection and the browser continued the execution,
+      // we just update the state back to "EXECUTING" so that it matches the current state.
```

Comment:
Maybe
```
// Lost socket connect, but browser continued to execute.
```
The rest is said in the code.

Reply:
Also updated the log message to be more clear. This spares us the comment.